uiowa/uiowa
# Move LB styles to their own library, override smooth scrolling

Related to #5441 but does not completely resolve it. Includes a refactoring of the code that sets a variable indicating whether LB is active for fragments and nodes.

Resolves https://github.com/uiowa/uiowa/issues/5441. 

# How to test
- `yarn workspace uids_base build`
- `ddev blt ds`
- `ddev drush @default.local uli /`
- Use inspector to check that `layout-builder-enabled` class is added to the `body` element.
- Use dev tools Network tab to check that `layout-builder.css` file _is not_ loaded.
- Click the "Layout" tab.
- Use dev tools Network tab to check that `layout-builder.css` file _is_ loaded.
- Visit https://default.uiowa.ddev.site/news and confirm that `layout-builder-disabled` class is added to the `body` element.
